### DISCUSSION TITLE
Support WASM plugins

### DISCUSSION THREAD

**Discussion by rholshausen** [pact-foundation/pact-plugins#85] [Ideas]
This is the discussion channel for the Support WASM plugins proposal.

[Proposal doc](https://github.com/pact-foundation/pact-plugins/blob/main/docs/proposals/003_Support_WASM_plugins.md)

**Comment by YOU54F** [comment]
I think we should convert this to an RFC in order to get more traction, same applies to 

- https://github.com/pact-foundation/pact-plugins/discussions/84
- https://github.com/pact-foundation/pact-plugins/discussions/83

What do you think @rholshausen I am happy to do so!
